_delphi_utils_python/delphi_utils/validator/datafetcher.py

```diff
@@ -3,18 +3,45 @@
 
 import re
 import threading
+import warnings
 from os import listdir
 from os.path import isfile, join
-import warnings
-import requests
-import pandas as pd
+from typing import Union
+
 import numpy as np
-import covidcast
+import pandas as pd
+import requests
+from delphi_epidata import Epidata
+from epiweeks import Week
+
 from .errors import APIDataFetchError, ValidationFailure
 
 FILENAME_REGEX = re.compile(
     r'^(?P<date>\d{8})_(?P<geo_type>\w+?)_(?P<signal>\w+)\.csv$')
 
+def _parse_datetimes(date_int: int, time_type: str, date_format: str = "%Y%m%d") -> Union[pd.Timestamp, None]:
```

Comment:
we don't need to have this function just yet since the validator only support day time type, but it was in the backlog of eventually supporting week time type, should I keep this or not?

Reply:
I think we can get rid of it because:
- it's only used by metadata fetching to parse min_time, max_time, and last_update columns from metadata and those columns aren't even used (we only use `signal` and `geo_type` for each source)
- supporting the week time type isn't easy to do and should probably be a separate PR